CONTRIBUTING.md

```diff
@@ -0,0 +1,80 @@
+# Contributing
+
+Thank you for thinking about contributing to TestPress! If you're unsure of anything, know that you're 💯 welcome to submit an issue or pull request on any topic. The worst that can happen is that you'll be politely directed to the best location to ask your question or to change something in your pull request. We appreciate any sort of contribution and don't want a wall of rules to get in the way of that.
+
+As with all WordPress projects, we want to ensure a welcoming environment for everyone. With that in mind, all contributors are expected to follow our [Code of Conduct](/CODE_OF_CONDUCT.md).
+
+All WordPress projects are [licensed under the GPLv2+](/LICENSE.md), and all contributions to TestPress will be released under the GPLv2+ license. You maintain copyright over any contribution you make, and by submitting a pull request, you are agreeing to release that contribution under the GPLv2+ license.
+
+This document covers the technical details around setup, and submitting your contribution to the TestPress project.
+
+## Getting Started
+
+TestPress is an Electron application, built primarily in JavaScript.
+
+To get started, you will need to [Node.js installed](https://nodejs.org/en/). Most MacOS and Linux systems will have Node already installed, but you may need to download and install it on Windows.
+
+TestPress can sometimes need a different version of Node than you have on your system, you can use [nvm](https://github.com/creationix/nvm) to change node versions on the command line:
+
+```
+npx nvm install
```

Comment:
Tested locally, and running nvm via npx didn't work.

```suggestion
nvm install
```

Reply:
Huh, interesting. We should probably fix the Gutenberg docs, too. 🙂